### PR TITLE
README: Add deps for translations and hardware wallet support

### PR DESCRIPTION
**Pull request by glv2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,26 +171,31 @@ sources are also used for statically-linked builds because distribution
...
+| GTest        | 1.5           | YES      | `libgtest-dev`[1]    | `gtest`      | `gtest-devel`       | YES      | Test suite      |
+| Doxygen      | any           | NO       | `doxygen`            | `doxygen`    | `doxygen`           | YES      | Documentation   |
+| Graphviz     | any           | NO       | `graphviz`           | `graphviz`   | `graphviz`          | YES      | Documentation   |
+| lrelease     | ?             | NO       | `qttools5-dev-tools` | `qt5-tools`  | `qt5-linguist`      | YES      | Translations    |
+| libhidapi    | ?             | NO       | `libhidapi-dev`      | `hidapi`     | `hidapi-devel`      | YES      | Hardware wallet |
+| libusb       | ?             | NO       | `libusb-dev`         | `libusb`     | `libusb-devel`      | YES      | Hardware wallet |
+| libprotobuf  | ?             | NO       | `libprotobuf-dev`    | `protobuf`   | `protobuf-devel`    | YES      | Hardware wallet |
+| protoc       | ?             | NO       | `protobuf-compiler`  | `protobuf`   | `protobuf-compiler` | YES      | Hardware wallet |
```
Comment (moneromooo-monero) [inline README.md]: What is this huge diff for ? It's hard to see what changed.

Comment (glv2) [reply README.md]: The main change is the 5 new lines at the end of the table. However some columns had to be made wider, so I realigned the other lines to keep the table as easy to read as it was.

Comment (moneromooo-monero) [reply README.md]: OK, my eyes glazed over and I missed them :)

Comment (hyc) [reply README.md]: Whitespace-only changes should be kept to a separate commit, so that changes like this don't get lost in the noise.

**Comment by dEBRUYNE-1** [comment]
@glv2 - Could you rebase this PR?

**Comment by glv2** [comment]
Rebased

**Comment by dEBRUYNE-1** [comment]
Thanks

File: README.md (hunk 1 above)
Comment (ndorf) [reply README.md]: The `-w` option to the `git diff` and `git show` commands will ignore the whitespace-only changes.

Comment (ndorf) [reply README.md]: Or, in the github interface, click the gear icon on e.g. https://github.com/monero-project/monero/pull/5896/files -- there's a "Hide whitespace" checkbox under there.

**Pull request merged**